appinventor/appengine/war/static/css/Ya.css

```diff
@@ -1199,7 +1212,7 @@ div.StatusPanel {
 }
 
 .ode-ChoicePropertyEditor:focus {
-  border-color: #D3D3D3;
+  border-color: rgba(82, 168, 236, 0.8);
```

Comment:
The colors here are now overwritten by variables in client/variableColors.css and client/style/neo/neo.css, and the variables are defined in light.css and dark.css in the same folders. The colors defined here now a fall-through for anything not defined using the color variables.

Your focus colors should be added to those css files, possibly including a new color variable. We can play around with the exact color ourselves if we think it needs adjustment.